docs/source/distributed.md

```diff
@@ -224,6 +224,10 @@ inconsistent 'UUID' assignment across ranks, and to prevent races during initial
 .. autofunction:: is_torchelastic_launched
 ```
 
+```{eval-rst}
+.. autofunction:: get_default_backend_for_device
```

Comment:
Please ignore this comment if CI pass.
```suggestion
.. autofunction:: torch.distributed.distributed_c10d.get_default_backend_for_device
```

Reply:
There is such a prefix (`torch.distributed.distributed_c10d.`) for few functions, but `init_process_group` and few others are defined in the same file as `get_default_backend_for_device` and don't have it. Plus `make html` did work for me locally and generated the correct .html output. I will monitor the CI and and if something will fail I'll make  adjustments.

Reply:
Thanks. `init_process_group` was ignored to check its "full name" in this [list](https://github.com/pytorch/pytorch/blob/568ca89bac9a80a66d664593a26ae69ac604796e/docs/source/conf.py#L604). Anyway, it is weird that PyTorch doc CI doesn't detect the missing API `get_default_backend_for_device`.